tools/mavlogdump.py

```diff
@@ -99,10 +100,10 @@
 if nottypes is not None:
     nottypes = nottypes.split(',')
 
-ext = os.path.splitext(filename)[1]
-isbin = ext in ['.bin', '.BIN', '.px4log']
-islog = ext in ['.log', '.LOG'] # NOTE: "islog" does not mean a tlog
-istlog = ext in ['.tlog', '.TLOG']
+mlog_type = str(type(mlog))
```

Comment:
Shouldn't this use `isinstance`?

Reply:
Yeah, I thought of using isinstance first. It does feel like nicer coding and more precise. The only thing that made me use str(type(x)) instead was that I would need to add an import (DFReader).
Happy to change it if that is the preferred etiquette.

Reply:
Code updated to use "isinstance".